recipe/meta.yaml

```diff
@@ -1,4 +1,5 @@
 {% set name = "osmnx" %}
+{% set base = "osmnx-base" %}
```

Comment:
The name is used in the URL multiple times but this one only once. Let's just hardcode it there to make the recipe smaller and clearer.